data/hai/hai missions.txt

```diff
@@ -1742,3 +1742,446 @@ mission "Nanachi Meeting"
+				accept
+
+event "pond strider mass production"
+	shipyard "Hai Advanced"
+		"Pond Strider"
+		"Flea"
+	shipyard "Hai Basics"
+		"Flea"
+	outfitter "Hai Basic"
+		"Railgun Slug"
+	outfitter "Hai Intermediate"
+		"Railgun"
+		"Railgun Slug"
+		"Railgun Slug Rack"
+		"Sand Cell"
+		`"Baellie" Atomic Engines`
+	outfitter "Hai Advanced"
+		"Railgun"
+		"Railgun Slug"
+		"Railgun Slug Rack"
+		"Sand Cell"
+		`"Baellie" Atomic Engines`
+	fleet "Large Hai"
+		add variant 1
+			"Pond Strider" 4
+			"Flea" 32
+			"Geocoris"
+		add variant 1
+			"Centipede (Pulse)" 3
+			"Centipede" 2
+			"Pond Strider" 3
+			"Flea" 24
+		add variant 1
+			"Pond Strider (Railgun)" 3
+			"Flea (Pulse)" 24
+			"Aphid" 10
+		add variant 2
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)" 2
+		add variant 2
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Lightning Bug" 2
+		add variant 1
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)"
+			"Aphid" 4
+		add variant 1
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+			"Lightning Bug"
+			"Aphid (Anti-Missile)" 4
+		add variant 2
+			"Pond Strider" 3
+			"Flea (Pulse)" 24
+			"Lightning Bug (Pulse)"
+			"Water Bug"
+			"Water Bug (Pulse)"
+		add variant 3
+			"Pond Strider" 3
+			"Flea (Pulse)" 12
+			"Flea" 12
+			"Water Bug"
+			"Water Bug (Pulse)"
+		add variant 2
+			"Pond Strider" 3
+			"Flea" 24
+			"Lightning Bug" 2
+			"Water Bug"
+		add variant 3
+			"Pond Strider" 3
+			"Flea" 24
+			"Lightning Bug (Pulse)" 2
+		add variant 2
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Shield Beetle"
+		add variant 2
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+		add variant 1
+			"Pond Strider (Ion)" 3
+			"Flea" 24
+			"Shield Beetle"
+			"Lightning Bug (Pulse)" 2
+		add variant 1
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+			"Lightning Bug" 2
+		add variant 1
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+			"Centipede" 3
+		add variant 2
+			"Centipede"
+			"Shield Beetle (Pulse)" 2
+			"Pond Strider (Railgun)" 3
+			"Flea" 24
+		add variant 1
+			"Geocoris"
+			"Shield Beetle (Pulse)"
+			"Pond Strider (Tracker)" 3
+			"Flea" 24
+		add variant 1
+			"Geocoris"
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Grasshopper"
+			"Grasshopper (Tracker)"
+		add variant 1
+			"Geocoris"
+			"Pond Strider (Pulse)" 3
+			"Flea" 24
+			"Grasshopper"
+			"Grasshopper (Tracker)"
+		add variant 1
+			"Geocoris (Tracker)"
+			"Aphid (Anti-Missile)" 2
+			"Aphid"
+			"Pond Strider" 3
+			"Flea" 24
+
+mission "Strider 4"
+	landing
+	source "Hai-home"
+	to offer
+		or
+			has "Strider 3 (Remnant): done"
+			has "Strider 3 (Coalition): done"
+	on offer
+		event "pond strider mass production" 30
+		conversation
+			``
+				decline
```

Comment:
This doesn't need to exist. Simply put the `event "pond strider mass production" 30` in the `on complete` section of `Strider 3 (Coalition)` and `Strider 4 (Remnant)`. It will be simpler, and doesn't require an empty conversation, either.

```suggestion
```

Reply:
You misunderstand the purpose of this mission. The Coalition and Remnant branches are (or were) suppose to both converge on this final mission where the Hai make comment about the technology discovered and tell you when the new Pond Strider will be ready. Notice how the Coalition branch currently has nothing happen when you return to Hai space; that's what this mission is suppose to be (hence the empty conversation and me saying earlier that this isn't finished).

Reply:
Ah, OK. That makes sense, then. I missed that comment somewhere. I'll delete the suggestion part of the comment to save some space and remove the distraction.